app/src/main/java/com/android/streetworkapp/model/image/ImageRepositoryFirestore.kt

```diff
@@ -255,6 +255,11 @@ class ImageRepositoryFirestore(
     }
   }
 
+  /**
+   * Deletes all the images related to a user
+   *
+   * @param userId The user id to whom we delete all the data from. (pictures uploaded and ratings)
```

Comment:
I would remove the text in parenthesis from the parameter field and add the fact that it also removes the ratings in the first sentence of the documentation, what do you think?

Reply:
agreed, changed it